### PR TITLE
BEFOUND-1702-create-repo-for-golden-samples

### PR DESCRIPTION
**Pull request by mostafarastgar**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    <properties>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```
Comment (ahrytsiuk) [inline pom.xml:32]: Are we inspiring to use `Lombok`? 😱

File: token-management/pom.xml [hunk 2]
```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.backbase.goldensample</groupId>
+        <artifactId>backend-devkit-golden-sample</artifactId>
+        <version>1.0.0</version>
```
Comment (ahrytsiuk) [inline token-management/pom.xml:9]: For me will be better to define `service-sdk-starter-core` as a parent for submodules. 

And to completely remove `pom.xml` from the root directory.

**Review by ahrytsiuk — APPROVED** [review]

File: token-management/src/main/java/com/backbase/goldensample/token/Application.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.backbase.goldensample.token;
+
+import com.backbase.audit.client.EnableAuditClient;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+
+@SpringBootApplication
+@EnableAuditClient
```
[the remark this answers is not in the document]
Comment (mostafarastgar) [reply token-management/src/main/java/com/backbase/goldensample/token/Application.java:10]: as you said extra condition doesn't look good. So I will leave it for the team to decide

File: token-management/src/main/java/com/backbase/goldensample/token/Application.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+@EnableDiscoveryClient
+public class Application extends SpringBootServletInitializer {
```
[the remark this answers is not in the document]
Comment (mostafarastgar) [reply token-management/src/main/java/com/backbase/goldensample/token/Application.java:12]: Nice catch. that came from plugin. I will try to fix it.

File: pom.xml (hunk 1 above)
Comment (mostafarastgar) [reply pom.xml:32]: what is the pitfall with that?

**Pull request merged**